db/migrate/20141015173220_add_docker_image_fields.rb

```diff
@@ -1,5 +1,6 @@
 class AddDockerImageFields < ActiveRecord::Migration[4.2]
   def up
+    return unless Katello.with_docker?
```

Comment:
I know this is still WIP but I don't think this is the right approach, the migration will be marked as having been applied, meaning you can never go back and add foreman-docker to a Katello instance later.